module/mailer/src/Resources/config/services.yml

```diff
@@ -0,0 +1,25 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ergonode\Mailer\Application\:
+        resource: '../../Application/*'
+
+    Ergonode\Mailer\Application\Command\:
+        resource: '../../Application/Command/*'
+        tags: [ 'console.command' ]
+
+    Ergonode\Mailer\Infrastructure\:
+        resource: '../../Infrastructure/*'
+        exclude: '../../Infrastructure/{Message}'
+
+    Ergonode\Mailer\Infrastructure\Handler\:
+        resource: '../../Infrastructure/Handler/*'
+        tags: [ 'messenger.message_handler' ]
```

Comment:
Probably this as well but not really sure. Worth verifying.

Reply:
Nope, it's throwing exception because can't find handlers for messages